source/ui/SXRMB/SXRMBScanConfigurationView.cpp

```diff
@@ -30,7 +30,7 @@ void SXRMBScanConfigurationView::updateFluorescenceDetectorComboBox(SXRMB::Fluor
 QComboBox *SXRMBScanConfigurationView::createFluorescenceComboBox()
 {
 	QComboBox *newComboBox = new QComboBox;
-	newComboBox->insertItem(0, "None");
+	newComboBox->insertItem(0, "Transmission");
```

Comment:
@dretrex I will change in your suggestion. Change "None" to "Transmission" is what Quanfeng ased.